cmd/pessimism/main.go

```diff
@@ -97,15 +97,32 @@ func main() {
 		os.Exit(1)
 	}
 
-	pess := app.New(ctx, m, srver)
+	pessimism := app.New(ctx, cfg, m, srver)
```

Comment:
why not server

Reply:
Because the http server is a field of the application